client/store/product.js

```diff
@@ -9,46 +9,47 @@ const UPDATE_PRODUCT = 'UPDATE_PRODUCT'
 const DELETE_PRODUCT = 'DELETE_PRODUCT'
 
 const productState = {
-  allProducts = [],
+  allProducts: []
```

Comment:
This is the primary change. The other changes below are Prettier's reformatting.